src/Stuart/Infrastructure/ApiResponse.php

```diff
@@ -20,7 +20,7 @@ public function getBody()
 
     public function success()
     {
-        return in_array($this->statusCode, [200, 201], true);
```

Comment:
I would add `204` to the array instead. It's not likely to change in the future and it's closer to what we document in our api documentation.